Tasks/Krupienkova/Task3/task3.py

```diff
@@ -0,0 +1,85 @@
+from datetime import datetime
+
+HOURS = {
+    1:  ['один час', 'второго', 'два'],
+    2:  ['два часа ', 'третьего', 'три'],
+    3:  ['три часа', 'четвертого', 'четыре'],
+    4:  ['четыре часа', 'пятого', 'пять'],
+    5:  ['пять часов', 'шестого', 'шесть'],
+    6:  ['шесть часов', 'седьмого', 'семь'],
+    7:  ['семь часов', 'восьмого', 'восемь'],
+    8:  ['восемь часов', 'девятого', 'девять'],
+    9:  ['девять часов', 'десятого', 'десять'],
+    10: ['десять часов', 'одинадцатого', 'одинадцать'],
+    11: ['одинадцать часов', 'двенадцатого', 'двенадцать'],
+    12: ['двенадцать часов', 'первого', 'час'],
+}
+
+MINUTES = {
+    0: 'ровно', 1: 'одна минута',
+    2: 'две минуты', 3: 'три минуты',
+    4: 'четыре минуты', 5: 'пять минут',
+    6: 'шесть минут', 7: 'семь минут',
+    8: 'восемь минут', 9: 'девять минут',
+    10: 'десять минут', 11: 'одинадцать минут',
+    12: 'двенадцать минут', 13: 'тринадцать минут',
+    14: 'четырнадцать минут', 15: 'пятнадцать минут',
+    16: 'шестнадцать минут', 17: 'семнадцать минут',
+    18: 'восемнадцать минут', 19: 'девятнадцать минут',
+    20: 'двадцать минут', 21: 'двадцать одна минута',
+    22: 'двадцать две минуты', 23: 'двадцать три минуты',
+    24: 'двадцать четыре минуты', 25: 'двадцать пять минут',
+    26: 'двадцать шесть минут', 27: 'двадцать семь минут',
+    28: 'двадцать восемь минут', 29: 'двадцать девять минут',
+    30: 'половина', 31: 'тридцать одна минута',
+    32: 'тридцать две минуты', 33: 'тридцать три минуты',
+    34: 'тридцать четыре минуты', 35: 'тридцать пять минут',
+    36: 'тридцать шесть минут', 37: 'тридцать семь минут',
+    38: 'тридцать восемь минут', 39: 'тридцать девять минут',
+    40: 'сорок минут', 41: 'сорок одна минута',
+    42: 'сорок две минуты', 43: 'сорок три минуты',
+    44: 'сорок четыре минуты', 45: 'без пятнадцати минут',
+    46: 'без четырнадцати минут', 47: 'без тринадцати минут',
+    48: 'без двенадцати минут', 49: 'без одинадцати минут',
+    50: 'без десяти минут', 51: 'без девяти минут',
+    52: 'без восьми минут', 53: 'без семи минут',
+    54: 'без шести минут', 55: 'без пяти минут',
+    56: 'без четырех минут', 57: 'без трех минут',
+    58: 'без двух минут', 59: 'без одной минуты',
+}
```

Comment:
the data structure is not optimized, the is a lot of duplicates

Reply:
This structure allows me to find correct answer by the cost of 2 dict lookups. Isn't it faster than 2 dict lookups **+ bunch of if/else**?  I suppose my structure is less optimal by memory consumption, but more optimal by performance than if-else.